lib/ssh.js

```diff
@@ -191,13 +193,20 @@ SSH.prototype.start = function(options) {
             readyTimeout: self.timeout
         });
     } else if (!self.pass && self.key) {
-        self._c.connect({
-            host: self.host,
-            port: self.port || 22,
-            username: self.user,
-            privateKey: self.key,
-            readyTimeout: self.timeout
-        });
+        try {
```

Comment:
What you couldve done is if both 'self.pass' and 'self.key' is available, have 'self.pass' be the passphrase.